driver_scripts/setup_MOC_basins.py

```diff
@@ -19,6 +19,8 @@ def spcall(args): #{{{
 
 options, args = parser.parse_args()
 
+groupNameMOC = 'MOCRegionsGroup'
+groupNameBasins = 'BasinRegionsGroup'
```

Comment:
To summarize:

 1. `groupNames` should be unique from each script
 2. I would include elements like `MOC`, `Global`, etc. as well as `Regions` and `Transects` for greater specificity in `groupNames`